Scripts/RemediationScripts/Remediate-SetDBForPostgreSQLSingleServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,847 @@
+<###
+# Overview:
+    This script is used to set minimium required TLS version and enable ssl enforcement for Azure Database for PostgreSQL single server in a Subscription.
+# Control ID:
+    Azure_DBforPostgreSQL_DP_Use_Secure_TLS_Version
+# Display Name:
+    Use approved version of TLS for Azure Database for PostgreSQL - single Server.
+    
+# Prerequisites:    
+    Contributor or higher priviliged role on the Azure Database for PostgreSQL single server(s) is required for remediation.
+# Steps performed by the script:
+    To remediate:
+        1. Validating and installing the modules required to run the script and validating the user.
+        2. Get the list of Azure Database for PostgreSQL single server(s) in a Subscription that have server parameter minimaltlsversion set as versions less than minimum required TLS version or parameter SslEnforcement is set to OFF .
+        3. Back up details of Azure Database for PostgreSQL single server(s) that are to be remediated.
+        4. Set supported minimum required TLS version by updating parameter minimaltlsversion as minimum required TLS version and enable SslEnforcement by updating server parameter SslEnforcement to ON for Azure Database for PostgreSQL single server(s).
+    To roll back:
+        1. Validate and install the modules required to run the script and validating the user.
+        2. Get the list of Azure Database for PostgreSQL single server(s) in a Subscription, the changes made to which previously, are to be rolled back.
+        3. Set the server parameter minimaltlsversion to original value and set server parameter SslEnforcement to ON as per input file.
```

Comment:
In rollback why we will set - set server parameter SslEnforcement to ON

Reply:
fixed.